### PR TITLE
[explorer] Add object video support

### PR DESCRIPTION
**Pull request by Jordan-Mysten**
## Description 

This adds video support to object displays in explorer. Some notes:
- The object display must have `video_url` property, _in addition_ to an `image_url` field. The `image_url` is expected to be the thumbnail of the video, and the fallback display in the event that videos are not supported. This could change in the future, but for now seems like a sensible approach.
- Only supported for recognized packages.

## Test Plan 

Ran locally.

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: apps/explorer/src/hooks/useRecognizedPackages.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+export function useRecognizedPackages() {
+    const [network] = useNetwork();
+
+    const recognizedPackages = useFeatureValue(
+        'recognized-packages',
+        DEFAULT_RECOGNIZED_PACKAGES
+    );
+
+    // Our recognized package list is currently only available on mainnet
```
Comment (williamrobertson13) [inline apps/explorer/src/hooks/useRecognizedPackages.ts:20]: would it be better to do this network gating via Growthbook attributes?

**Review by williamrobertson13 — APPROVED** [review]
very cool!

File: apps/explorer/src/components/OwnedObjects/OwnedObject.tsx [hunk 2]
```diff
@@ -12,14 +13,17 @@ type OwnedObjectTypes = {
...
     const displayMeta = getObjectDisplay(obj).data;
+
     return (
         <ObjectDetails
             variant="small"
             id={obj.data?.objectId}
             type={trimStdLibPrefix(parseObjectType(obj))}
             name={displayMeta?.name ?? displayMeta?.description}
             image={displayMeta?.image_url}
+            video={video}
```
Comment (plam-ml) [inline apps/explorer/src/components/OwnedObjects/OwnedObject.tsx:26]: I saw the `desc` says the image_url must be present. Do we check for that anywhere? I only see the `image` being optional in `ObjectDetails`.

**Review by plam-ml — APPROVED** [review]

File: apps/explorer/src/components/OwnedObjects/OwnedObject.tsx (hunk 2 above)
Comment (Jordan-Mysten) [reply apps/explorer/src/components/OwnedObjects/OwnedObject.tsx:26]: Yeah this component is only rendered when there's an image url that's resolved, this is done at a higher level

**Pull request merged**